ehrapy/api/preprocessing/_normalization.py

```diff
@@ -219,6 +226,25 @@ def _norm_power_box_cox(values: np.ndarray) -> np.ndarray:
         return power_transform(values, method="box-cox")
 
 
+def _norm_log1p(values: np.ndarray, base: int | float | None) -> np.ndarray:
```

Comment:
What do you think about having a default value for this normalization? If we set one we should also document it.

Reply:
For the base? The default will be natural log if `base = None` which I think I did in the master function but not here I guess. Could add it here as well.

Reply:
Yes, for the base. I couldn't see a default in the master function. Maybe I am blind?

Reply:
Here (L25) after all the `|` (which I think is the right syntax) 

```python
def normalize(
    adata: AnnData, methods: dict[str, list[str]] | str, base: int | float | None = None, copy: bool = False
)
```

Reply:
Ah yeah, think we misunderstood each other. Yes, internally the default will be set to the natural log, but this might not be visible or known to the user. If we make it explicit here it is more clear. We can also just document this behavior.

Not a big issue.